src/options.ts

```diff
@@ -93,6 +93,12 @@ export interface InternalRoomOptions {
   e2ee?: E2EEOptions;
 
   loggerName?: string;
+
+  /**
+   * shorten the publish latency by establish publisher peerconnection when joining room, will
+   * cost few cpu time & network traffic even if there is no track has been published.
+   */
+  fastPublish: boolean;
```

Comment:
this should not be a required option to set
```suggestion
  fastPublish?: boolean;
```